### PR TITLE
Multi Step Form Added

### PR DESCRIPTION
**Pull request by Prasanna0908**
Multi-Step Form Template Added

### CODE REVIEW

**Comment by ekhiw** [comment]
I think you should create new file in `src/data/templates/` and fill it with you template name and template file name, then import into `src/data/index.tsx` add new children in Forms.

**Comment by Prasanna0908** [comment]
Made the required changes.

File: tsconfig.json [hunk 1]
```diff
@@ -20,6 +20,6 @@
       "@/layout/*": ["layout/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/pages/templates/forms/multi-step/multiStep.tsx"],
```
Comment (hauptrolle) [inline tsconfig.json:23]: This should not be neccessary.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "codetogether.virtualCursorJoin": "sharedVirtualCursor"
+}
```
Comment (hauptrolle) [inline .vscode/settings.json:3]: Please remove that file

**Review by hauptrolle — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution. I have left some comments.

**Comment by Prasanna0908** [comment]
Made the required changes. Can you check once @hauptrolle

File: .vscode/settings.json (hunk 2 above)
Comment (Prasanna0908) [reply .vscode/settings.json]: Done

**Comment by hauptrolle** [comment]
@Prasanna0908 We are still getting errors on the build step. Are all the paths correct?

```
Error occurred prerendering page "/forms/multiStep". Read more: https://nextjs.org/docs/messages/prerender-error
--
10:50:30.594 | Error: Cannot find module './forms/multiStep/multistep'
``

**Comment by hauptrolle** [comment]
Can I fix the filename reference? There should be a capital "S".

**Comment by Prasanna0908** [comment]
> Can you fix the filename reference? There should be a capital "S".

Yes, please !

**Comment by Prasanna0908** [comment]
@hauptrolle Any update?

**Comment by hauptrolle** [comment]
> @hauptrolle Any update?

Can you fix this? I can't edit files from your PR.

**Comment by Prasanna0908** [comment]
is this the correct path (with a capital S)?
'./forms/multiStep/multiStep'

**Comment by hauptrolle** [comment]
> is this the correct path (with a capital S)? './forms/multiStep/multiStep'

Lets streamline the overall naming a little bit:

The content of `src/data/templates/multistep.ts` should be:

```
import { SubCategory } from '../types';

export const multistep: SubCategory = {
  name: 'Multi Step Forms',
  id: 'multistep',
  children: [
    {
      name: 'Basic 3 Step Form',
      filename: 'basic3StepForm',
    },

  ],
};
```

`src/pages/templates/forms/multi-step/multiStep.tsx` should be `src/pages/templates/forms/multistep/basic3StepForm.tsx`

**Comment by Prasanna0908** [comment]
Hey, @hauptrolle I made the required change. Can you please check once?

**Comment by hauptrolle** [comment]
Please read my comment again. You renamed the wrong file 😬

**Comment by Prasanna0908** [comment]
> Please read my comment again. You renamed the wrong file 😬

Is it possible to join a google meet? It would be convenient

**Comment by hauptrolle** [comment]
> > Please read my comment again. You renamed the wrong file 😬
> 
> Is it possible to join a google meet? It would be convenient

Nope. Please read my comment. You can test it locally by running `yarn build` and check if the build succeeds.

**Comment by Prasanna0908** [comment]
I tried everything as mentioned but it is still giving build error
image

**Comment by ekhiw** [comment]
@Prasanna0908 i create PR to your fork, maybe i can help

**Comment by hauptrolle** [comment]
> I tried everything as mentioned but it is still giving build error image

Have you read the contribution guide? Looks like your Github token is missing ...
https://github.com/hauptrolle/chakra-templates/blob/main/CONTRIBUTING.md

**Comment by Prasanna0908** [comment]
Yes, I have provided github token. Also, `yarn build` command is not working on the main branch also. Is it an issue with the previous commit by any chance?

**Comment by hauptrolle** [comment]
Maybe we should close this PR here, and you create a new one with the file structure I mentioned in my comment earlier?

**Comment by Prasanna0908** [comment]
> Maybe we should close this PR here, and you create a new one with the file structure I mentioned in my comment earlier?

I tried that as well. The thing is yarn build is not working on main branch also. Please have a look at it

**Comment by ekhiw** [comment]
@Prasanna0908 please check my pull request to your fork, im fixing file structure and typo.

**Comment by Prasanna0908** [comment]
I think all checks have passed now. Now check once @hauptrolle . Thanks @ekhiw

**Review by hauptrolle — APPROVED** [review]

**Pull request merged**